.github/workflows/pull-request.yml

```diff
@@ -37,29 +35,12 @@ jobs:
               - '.github/actions/setup-job-docker/*'
```

Comment:
❓ in the line above there is a filter for `'.github/workflows/ci*'`, but there are no workflows anymore which start with `ci`. we should either remove or update that filter.